dev/bin/knative-installer.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
is this file executable?

Reply:
[It seems to be](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/583/files#diff-0facce3621ef7b3815a47809a968cb4d1dafc823dbaea4f1ab136788e6dea998R118).

Reply:
...and [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/583/files#diff-4f22e85ef97a87798146e10c5677e5f832e5bfd91ee07148f85bc1e91331e3a0R50).